Source/WebInspectorUI/UserInterface/Views/ConsoleMessageView.js

```diff
@@ -159,6 +164,43 @@ WI.ConsoleMessageView = class ConsoleMessageView extends WI.Object
         this._repeatCountElement.textContent = Number.abbreviate(count);
     }
 
+    set timestamp(timestamp)
+    {
+        this._message._timestamp = timestamp;
+        if (this._element)
+            this._renderTimestamp(true);
+    }
+
+    _renderTimestamp(renderlatestTimestamp = false)
+    {
+        if (!this._message._timestamp) {
+            if (this._timestampElement) {
+                this._timestampElement.remove();
+                this._timestampElement = null;
+            }
```

Comment:
We can simplify the nesting of this by using optional chaining without the `if (this._timestampElement)` e.g.
```
this._timestampElement?.remove();
this._timestampElement = null;
```